<launch>

	<node pkg="mybot_control" name="rob_mover" type="rob_mover" output="screen"> <!--launch-prefix="xtermâˆ’e"-->
	</node>

</launch>
